src/common.js

```diff
@@ -118,7 +118,7 @@ function error(msg, _code, options) {
   state.error += logEntry;
 
   // Throw an error, or log the entry
-  if (config.fatal) throw new Error(logEntry);
+  if (config.fatal && options.fatal !== false) throw new Error(logEntry);
```

Comment:
Instead of this, how about we assign `fatal: config.fatal` up above in `DEFAULT_OPTIONS`?

Reply:
Fixed it; not sure how I missed that pattern originally.

Reply:
You should update this to check `options.fatal` instead of `config.fatal` (otherwise, this should be OK)